test/unit/oidc_jwt_token_validate_test.rb

```diff
@@ -0,0 +1,72 @@
+require 'test_helper'
+require 'ostruct'
+require 'openssl'
+require 'jwt'
+
+class OidcJwtTokenValidateTest < ActiveSupport::TestCase
+  context '#valid_jwt?' do
+    def setup
+      skip "SSO feature is not available for Ruby < 2.4.0" unless
```

Comment:
This is the same code but it can easily hide problems. I'd actually use `RUBY_VERSION` here so you know it's tested on newer rubies.